scripts/start-auto-claude.sh

```
@@ -0,0 +1,89 @@
+  exit 1
+fi
+
+if ! docker info > /dev/null 2>&1; then
+  log "${RED}❌ Docker is not running. Please start Docker Desktop.${NC}"
+  exit 1
+fi
+log "${GREEN}✓ Docker is running${NC}"
+
+# Step 2: Start Memory Layer (FalkorDB) in background
+log "${YELLOW}Starting Memory Layer (FalkorDB)...${NC}"
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)